flutter/lib/src/feedback/sentry_feedback_widget.dart

```diff
@@ -101,11 +112,12 @@ class _SentryFeedbackWidgetState extends State<SentryFeedbackWidget> {
                         controller: _nameController,
                         decoration: InputDecoration(
                           border: const OutlineInputBorder(),
-                          hintText: widget.namePlaceholder,
+                          hintText: widget.options.namePlaceholder,
                         ),
                         keyboardType: TextInputType.text,
                         validator: (String? value) {
```

Comment:
Doesn't the feedback text have a max length? That's probably a good addition. You can use [`LengthLimitingTextInputFormatter`](https://api.flutter.dev/flutter/services/LengthLimitingTextInputFormatter-class.html) for that.